Sources/XMTP/ApiClient.swift

```diff
@@ -71,14 +106,20 @@ class GRPCApiClient: ApiClient {
 			request.pagingInfo.cursor = cursor
 		}
 
-        var encodedPaging = ""
-        if request.hasPagingInfo {
-            encodedPaging = try request.pagingInfo.jsonString()
-        }
-        let responseJson = try await rustClient.query(topic: topic, json_paging_info: encodedPaging)
-        // Decode the response as a QueryResponse
-        let decodedQueryResponse = try QueryResponse(jsonString: responseJson)
-        return decodedQueryResponse
+		let paging = XMTPRust.PagingInfo(limit: 0, cursor: nil, direction: XMTPRust.SortDirection.Ascending)
+		let response = try await rustClient.query(topic.intoRustString(), Optional.none, Optional.none, Optional.none)
```

Comment:
Shouldn't we be passing in the `pagingInfo`, `startTime`, and `endTime` here?

Reply:
good catch!